### PR TITLE
fix: edit profile modal router

### PR DESCRIPTION
**Pull request by alantoa**
# Why

context: https://showtime-rq88331.slack.com/archives/C02Q0E4PBD0/p1680517764900479

# How

# Test Plan

### CODE REVIEW

**Review by intergalacticspacehighway — APPROVED** [review]
LGTM ❇️

**Review by hirbod — APPROVED** [review]
Thanks 🐐

File: packages/app/navigation/use-navigate-to.tsx [hunk 1]
```diff
@@ -31,8 +31,27 @@ export const useNavigateToLogin = () => {
 export const useNavigateToOnboarding = () => {
   const router = useRouter();
 
-  const navigateToOnboarding = () => {
-    router.push("/profile/onboarding");
+  const navigateToOnboarding = (replace?: boolean) => {
```
Comment (hirbod) [inline packages/app/navigation/use-navigate-to.tsx]:
```suggestion
  const navigateToOnboarding = ({ replace }: { replace?: boolean } ) => {
```

File: packages/app/hooks/use-user.ts [hunk 2]
```diff
@@ -22,11 +23,12 @@ export function useUser(params?: UserParams) {
       context?.isIncompletedProfile &&
       router
     ) {
-      router.replace("/profile/onboarding");
+      navigateToOnboarding(true);
```
Comment (hirbod) [inline packages/app/hooks/use-user.ts]:
```suggestion
      navigateToOnboarding({ replace: true });
```

**Review by hirbod — APPROVED** [review]
LGTM, but I personally prefer using object notation. It's more readable and can be extended easier in the future.

**Pull request merged**